src/views/LotteryV2/index.tsx

```diff
@@ -76,15 +71,14 @@ const LotteryV2 = () => {
           </Flex>
           <NextDrawCard />
         </Flex>
-      </TicketsSection>
-      <PageSection background={CHECK_PRIZES_BG} hasCurvedDivider={false} index={2}>
+      </PageSection>
+      <PageSection background={CHECK_PRIZES_BG} hasCurvedDivider={false} index={4}>
```

Comment:
Maybe the indexes can all be 0 or 1 with the exception of the top `Hero` section, which should be 1 higher?

Reply:
True. Actually they all can be 2 except Hero which needs to be lower, i.e. 1. Updated